packages/web-components/src/styles/patterns/button.ts

```diff
@@ -292,6 +292,7 @@ export const HypertextStyles = css`
     }
     :host(.hypertext) .control:${focusVisible} {
         border-bottom: calc(var(--focus-outline-width) * 1px) solid ${neutralFocusBehavior.var};
+        margin-bottom: calc(calc(var(--outline-width) - var(--focus-outline-width)) * 1px);
```

Comment:
Thanks @robarbms! You'll need to generate a change file by running `yarn change` in the root of the repo.

As an aside, I noticed in an example that when this is used *inline*, which is the prescribed usage, the height appears to follow the sizing of the back-plated instances (accent, neutral, etc). Ultimately this should sit inline with a paragraph (for example).

Reply:
Can you see if that's still a problem?

Reply:
I was able to confirm - just need the change file! 
![image](https://user-images.githubusercontent.com/13071055/105086782-5ed68c80-5a4e-11eb-8c2d-807a53ff8fd0.png)

Reply:
Added the change file.